src/axom/quest/MarchingCubesAlgo.cpp

```diff
@@ -0,0 +1,614 @@
+// Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include "axom/quest/MarchingCubesAlgo.hpp"
+#include "axom/quest/detail/marching_cubes_lookup.hpp"
+#include "axom/mint/mesh/UnstructuredMesh.hpp"
+#include "conduit_blueprint.hpp"
+#include "axom/fmt.hpp"
+
+#ifndef __WHERE
+  #define __STRINGIZE(x) __STRINGIZE2(x)
```

Comment:
why both stringize and strinize2 here?

Reply:
It's a weird consequence of the C macro features.  I don't think it can be done more simply.  Anyway, I should remove it.  It was just for my debugging convenience.